bundles/simple/plugin-2.go

```diff
@@ -0,0 +1,10 @@
+package main
```

Comment:
@imogenkraak I would suggest renaming the file like this: `plugin_2.go`. And you can replace `plugin.go` with `plugin_1.go`.